framework/src/modules/chain/submodules/transport.js

```diff
@@ -306,16 +306,17 @@ Transport.prototype.onUnconfirmedTransaction = function(
 	broadcast
 ) {
 	if (broadcast && !__private.broadcaster.maxRelays(transaction)) {
+		const transactionJSON = transaction.toJSON();
 		__private.broadcaster.enqueue(
 			{},
 			{
 				api: 'postTransactions',
 				data: {
-					transaction,
+					transaction: transactionJSON,
```

Comment:
TBH, I'm not so sure what we are fixing here. Anyways, in the case we do fix something, we should perform the same action in other places like in here https://github.com/LiskHQ/lisk-sdk/blob/61969851e484a18e119810851177f85091c3e774/framework/src/modules/chain/submodules/transport.js#L288

Reply:
signature is always an object so it should not need the toJSON